feat: Add Environment Variable module (#409)
starship/starship: docs/config/README.md

truncation_length = 8
```

## Environment Variable

The `env_var` module displays the current value of a selected environment variable.
The module will be shown only if any of the following conditions are met:

> - The environment variable named `variable` is defined

Reviewer: I think this may be misleading.
My first impression is that it would be required to have `$VARIABLE` be set.
Maybe instead it should be something along these lines:

```suggestion
- The `variable` configuration option matches an existing environment variable
```

Does this sound clear to folks?

- The environment variable named `variable` is not defined and `default` config value exists

### Options

| Variable   | Default          | Description                                                                  |
| ---------- | ---------------- | ---------------------------------------------------------------------------- |
| `symbol`   | `""`             | The symbol used before displaying the variable value.                        |
| `variable` | `""`             | The environment variable to be displayed.                                    |

> | `default`  | `""`             | The default value to be displayed when the selected variable is not defined. |

Reviewer 2: `symbol`, `variable`, `default` don't have a default value. Just leave it blank.

Author: This is fixed now

| `prefix`   | `""`             | Prefix to display immediately before the variable value.                     |
| `suffix`   | `""`             | Suffix to display immediately after the variable value.                      |
| `style`    | `"dimmed black"` | The style for the module.                                                    |
| `disabled` | `false`          | Disables the `env_var` module.                                               |

### Example

```toml
# ~/.config/starship.toml

[env_var]

> variable = "SHELL"

Reviewer: Can we throw in default to the example?
```suggestion
variable = "SHELL"
default = "unknown shell"
```

truncation_length = 8
```

## Environment Variable

The `env_var` module displays the current value of a selected environment variable.
The module will be shown only if any of the following conditions are met:

- The `variable` configuration option matches an existing environment variable

> - The environment variable named `variable` is not defined and `default` config value exists

Reviewer: I think this line should also receive the same treatment:
```suggestion
- The `variable` configuration option is not defined and the `default` configuration option is defined
```
